client/blocks/inline-help/inline-help-search-card.jsx

```diff
@@ -69,8 +74,29 @@ class InlineHelpSearchCard extends Component {
 				this.props.selectNextResult();
 				break;
 			case 'Enter': {
-				const hasSelection = this.props.selectedResultIndex >= 0;
-				hasSelection && this.props.openResult( event, this.props.selectedResult );
+				const { selectedResultIndex, selectedResult, query, onSelect, track } = this.props;
+
+				// check and catch admin section links.
+				const { support_type: supportType, link } = selectedResult;
+				if ( supportType === SUPPORT_TYPE_ADMIN_SECTION && link ) {
+					track( 'calypso_inlinehelp_admin_section_search', {
```

Comment:
Just thinking out loud here: is `calypso_inlinehelp_admin_section_search` a good descriptive event name here? In terms of the event we're performing it's not a search, right? Would something like `calypso_inlinehelp_admin_section_visit` or `calypso_inlinehelp_admin_section_open` be a better fit for actually visiting the section?

Reply:
Yes, for sure. Just FYI, the name was defined in the original [PR here](https://github.com/Automattic/wp-calypso/pull/31903/files#diff-e9a851d63ae8a38656bf41731802b1b3R140). Happy to update it. There is [an issue](https://github.com/Automattic/wp-calypso/issues/43843) where we can discuss the event name if it's the case.

Reply:
Changed the event name here 6d629e84fea31fe386f5a7bbfc02369103263e64